### ISSUE TITLE
Automation Idea

### ISSUE THREAD

**Issue by DeicideHD** [splitscreen/ss-twitter#11]
Suggestions

Automatically pull tweets and broadcast them

Spam Filters

Simple 10 second cooldown

10 second preview before it posts(So no obscene images make it through)

Image option On and Off obviously

and finally the most complex yet probably the best way to really implement the idea.

Use the twitch, twitter, youtube, and or other api to pull followers, subscribers, contributors etc really this gives the freedom to have the audience that is following and contributing to your cast to be seen and promoted. It also opens up things like new subscription perks that benefit the community and broadcaster. Your subs now get to be interactive, their Im live! tweets get seen and the entire community grows.

**Comment by techguydave** [comment]
This is definitely a great idea!

Right now my use case doesn't need it and it'll probably take quite a bit of planning and actual writing, but I would love to implement it at some point in time.

Side note, would a queue also work in junction with this? Maybe have stuff automatically added to a queue, but then display a panel where you can approve/disapprove individual tweets. While not completely automated, it's a lot easier than finding tweets and copying the links. Plus, you probably never want stuff from the masses thrown on-screen unfiltered anyway. Good way to get yourself in trouble! The queue could also be useful in manual mode, too.

**Comment by DeicideHD** [comment]
A queue system might be the best idea, just have it parse the 5-10 most
recent tweets every few minutes, and the take/post options as is w/o having
to get the url yourself. I still think the API for subs/followers etc would
be godly, but thats a big chunk of work regardless. Now the queue idea is
great and I think there is probably an easy solution tbh.

**Comment by techguydave** [comment]
Yeah, the thing about subs/followers is that you'd have to have some way to match the viewer to their Twitter account in order to verify that they're the same person. For instance, I may be "synth3tk" on Twitch but "JohnJacob" on Twitter. How to link those two is not trivial.

**Comment by DeicideHD** [comment]
simple login page requiring both, not clean but very effective
